### PR TITLE
[WWST-1613] Including fingerprint for Ozom Smart Plug

### PR DESCRIPTION
**Pull request by luisferreira1**
Change-Id: I8d4e7ef186e878923fe3d9589c2a38be26b390f4

### CODE REVIEW

**Comment by luisferreira1** [comment]
@mckeed @tpmanley @MAblewiczS @greens Please review my DTH.

**Comment by MAblewicz** [comment]
@greens @tpmanley luisferreira1 is a PE from SRBR (he doesn't have right to add reviewers so i did this istead).

**Review by MAblewicz — DISMISSED** [review]
Look ok. Approved

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 1]
```diff
@@ -27,6 +27,7 @@ metadata {
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702"
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "0003, 000A, 0019", manufacturer: "Jasco Products", model: "45853", deviceJoinName: "GE ZigBee Plug-In Switch"
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "000A, 0019", manufacturer: "Jasco Products", model: "45856", deviceJoinName: "GE ZigBee In-Wall Switch"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", manufacturer: "ClimaxTechnology", model: "PSM_00.00.00.35TC", deviceJoinName: "Ozom Smart Plug"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy]: Correct me if I'm wrong but I think this device also has a single out cluster `0000`. If that's true can you add `outClusters: "0000"` to the fingerprint?

Comment (MAblewicz) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy]: @tpmanley Maybe we should make it a standard to add RAW fingerprint as a comment?

Comment (luisferreira1) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy]: @tpmanley , you are right. I added outClusters: "0000" to fingerprint.
@MAblewiczS, I added RAW fingerprint as comment.
Can you check please?

**Review by MAblewicz — CHANGES_REQUESTED** [review]
@luisferreira1 We agreed with Tom that RAW fingeprint should be added in github comment (like when you ask reviewers to check) also based on you raw fingerprint Tom was right, You were  missing 0000 out cluster but it is in now. just remove comment  and we should be good to go.

**Comment by luisferreira1** [comment]
@MAblewiczS Changes done, and also added raw fingerprint below:
simple":"01 0104 0009 00 06 0000 0003 0004 0005 0006 0702 01 0000","application":"10","manufacturer":"ClimaxTechnology","model":"PSM_00.00.00.35TC"
@tpmanley @greens can you ckeck please?

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 2]
```diff
@@ -27,6 +27,8 @@ metadata {
...
```
Comment (MAblewicz) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy]: Remove this line. It next line is correct one.

**Review by MAblewicz — DISMISSED** [review]

**Review by tpmanley — APPROVED** [review]
Changes look good. Be sure to squash the commits when you merge.

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy (hunk 2 above)
Comment (luisferreira1) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy]: Ok, line is removed.

**Pull request merged**